src/_sass/_secondary_nav.scss

```diff
@@ -0,0 +1,196 @@
+$background-nav-color: #e76c7a;
+$secondary-nav-color: white;
+$current-color: white;
+
+.site-nav {
+  background-color: $background-nav-color;
+  height: 4.5rem;
+  position: fixed;
+  top: 4.5rem;
+  // transition: top cubic-bezier(0.71,-0.04, 0.32, 1.07) .2s;
+  width: 100%;
+  z-index: 999;
+  
+
+  .logo {
+    margin-top: 0.5625rem;
+    // height: 2.5rem;
+  }
+
+  .content {
+    align-items: center;
+    background-color: $background-nav-color;
+    display: flex;
+    float: none;
+    height: 100%;
+    justify-content: space-between;
+    padding: 0 1.25em;
+    z-index: 998;
+  }
+
+  .sub-nav-left {
+    align-items: center;
+    display: flex;
+  }
+
+  .up {
+    transform: rotate(180deg)
+  }
+
+  .down {
+    border-left: 0.5rem solid transparent;
+    border-right: 0.5rem solid transparent;
+    border-top: 0.5rem solid $secondary-nav-color;
+    height: 0; 
+    transition: all .2s;
+    width: 0; 
+  }
+
+  .sub-nav-control {
+    margin-left: 0.625rem;
+    transition: linear .2s;
+  }
+
+
+  .sub-nav-control:hover {
+    cursor: pointer;
+  }
+
+  .hello {
+    align-items: center;
+    display: flex;
+    height: 100%;
+    
+    // .secondary-nav-link {
+    //   color: $secondary-nav-color;
+    //   font-family: "Open Sans";
+    //   font-size: 1rem;
+    //   letter-spacing: -0.0125rem;
+    //   line-height: 1rem;
+    //   padding-bottom: 0.5rem;
+    //   text-decoration: none;
+    // }
+
+    .secondary-nav-link {
+      position: relative;
+      color: white;
+      text-decoration: none;
+    }
+    
+    .secondary-nav-link:before {
+      content: "";
+      position: absolute;
+      width: 100%;
+      height: 2px;
+      bottom: -10px;
+      left: 0;
+      background-color: white;
+      visibility: hidden;
+      -webkit-transform: scaleX(0);
+      transform: scaleX(0);
+      -webkit-transition: all 0.3s ease-in-out 0s;
+      transition: all 0.3s ease-in-out 0s;
+    }
+    
+    .secondary-nav-link:hover {
+      cursor: pointer;
+    }
+    
+    .secondary-nav-link:hover:before {
+        visibility: visible;
+        transform: scaleX(1);
+    }
+
+    ul {
+      display: flex;
+      justify-content: space-between;
+      li {
+        margin-left: 1.875rem;
+      }
+      .current:before {
+        visibility: visible;
+        transform: scaleX(1);
+      }
+    }
+    
+  }
+
+  .current-active {
+    color: $current-color;
+  }
+
+  .content:after {
```

Comment:
this should be inside or right after the .content styles at the top of the document